src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -19,6 +19,20 @@
             <argument /><!-- Cache-Directory -->
         </service>
 
+        <service id="annotations.opcache_warmer" class="Symfony\Bundle\FrameworkBundle\CacheWarmer\AnnotationsCacheWarmer" public="false">
+            <argument type="service" id="annotations.reader" />
+            <argument>%kernel.cache_dir%/annotations.php</argument>
+        </service>
+
+        <service id="annotations.cache_factory" class="Symfony\Bundle\FrameworkBundle\Cache\PhpCachePoolFactory" public="false">
+            <argument type="service" id="cache.pool.local" />
```

Comment:
we should use a dedicated `cache.pool.annotations` service instead of `cache.pool.local`